docs/scaling/kubernetes.txt

```diff
@@ -1,78 +1,210 @@
-==========
-Kubernetes
-==========
+.. _scaling-kube:
 
-Kubernetes is an open-source system for automating deployment, scaling, and
-management of containerized applications that builds on 15 years of experience
-running production workflows at Google.
+=============================
+Scaling CrateDB on Kubernetes
+=============================
 
-Thanks to our :ref:`Docker image <cratedb_docker>`, managing your CrateDB
-cluster with Kubernetes requires only a few steps.
+CrateDB and Docker are a great match thanks to CrateDB’s `horizontally
+scalable`_ `shared-nothing architecture`_ that lends itself well to
+`containerization`_.
 
-This guide will assume that you already have deployed a CrateDB cluster on
-Kubernetes, as detailed in our :ref:`Kubernetes Deployment <getting_started_kubernetes>` guide.
+`Kubernetes`_ is an open-source container orchestration system for the
+management, deployment, and scaling of containerized systems.
+
+Together, Docker and Kubernetes are a fantastic way to deploy and scale CrateDB.
+
+.. NOTE::
+
+    This guide assumes you have already deployed CrateDB on Kubernetes.
+
+    Consult the :ref:`Kubernetes deployment guide <getting_started_kubernetes>`
+    for more help.
+
+.. SEEALSO::
+
+    The official `CrateDB Docker image`_.
 
 .. rubric:: Table of Contents
 
 .. contents::
    :local:
 
-Scale CrateDB
-=============
+.. _scaling-kube-kube:
+
+Kubernetes Reconfiguration
+==========================
 
-You can scale your CrateDB cluster by scaling the number of replicas of the
-StatefulSet to the desired number of CrateDB nodes:
+You can scale your CrateDB cluster by increasing or decreasing the configured
+number of replica `pods`_ in your `StatefulSet`_ controller to the desired
+number of CrateDB nodes.
+
+.. _scaling-kube-command:
+
+Using an Imperative Command
+---------------------------
+
+You can issue an `imperative command`_ to make the configuration change, like
+so:
 
 .. code-block:: sh
 
-    kubectl scale statefulsets crate --replicas=4
+    sh$ kubectl scale statefulsets crate --replicas=4
+
+.. NOTE::
+
+    This makes it easy to scale quickly, but your cluster configuration is not
+    reflected in your `version control`_ system.
+
+.. _scaling-kube-vc:
 
-or by editing the value in the StatefulSet directly:
+Using Version Control
+---------------------
+
+If you want to version control your cluster configuration, you can edit the
+StatefulSet controller configuration file directly:
```

Comment:
Maybe worth mentioning here that the thing you're updating here is the `replicas` setting.